assets/blocks/quiz/quiz-block/use-has-questions.js

```diff
@@ -16,28 +16,39 @@ export const useHasQuestions = ( clientId ) => {
 			.filter( ( block ) => !! block.attributes.title )
 	);
 
+	const { editPost } = useDispatch( 'core/editor' );
+	const setQuizHasQuestions = useCallback(
+		( on ) => {
+			const { _quiz_has_questions: savedValue } =
+				selectData( 'core/editor' ).getCurrentPostAttribute( 'meta' ) ||
```

Comment:
I wonder if it'd more appropriate to use the `useSelect` for this and the other `useEffect` too. WDYT?

Reply:
Also included in 32b1714